tests/testthat/reporters/progress.txt

```diff
@@ -1,12 +1,5 @@
 ✔ | OK F W S | Context
 
-⠏ |  0       | Expectations
-⠋ |  1       | Expectations
-⠙ |  1 1     | Expectations
-⠹ |  1 2     | Expectations
-⠸ |  1 3     | Expectations
-⠼ |  1 4     | Expectations
-⠴ |  2 4     | Expectations
 ✖ |  2 4     | Expectations
```

Comment:
Do you think it is possible to keep these lines? Because otherwise the tests are not really testing the output. Maybe by setting `min_time`?

Timing based tests are not too great in general, but of course in this case we can't really avoid them. We can have another set of (the same?) tests that set `min_time` to something very large, so that the output is omitted on very slow systems as well.